src/sequence.rs

```diff
@@ -1,6 +1,5 @@
 use std::fmt;
-use std::sync::atomic::{AtomicU16, AtomicU64, Ordering};
-use std::sync::Arc;
+use std::sync::{Arc, Mutex};
```

Comment:
let's use util crate's sync feature (https://github.com/webrtc-rs/util/blob/main/Cargo.toml#L21)

https://github.com/webrtc-rs/util/blob/main/src/sync/mod.rs#L2

Reply:
@rainliu since this PR is closed, do you want me to open another pull request with suggested changes ?